app/controllers/proposals_controller.rb

```diff
@@ -12,7 +12,9 @@ class ProposalsController < ApplicationController
 
   def show
     @proposal = proposal.decorate
-    cookies[:detail] = params[:detail]
+    if !params[:detail].blank?
```

Comment:
@jessieay I had to re-add this check because the cookie was getting wiped when the query wasnt set.